### DISCUSSION TITLE
Latest Bootstrap version and default Wordpress jQuery 1.12.4-wp

### DISCUSSION THREAD

**Discussion by kanlukasz** [twbs/bootstrap#32228] [Q&A] [answered]
Wordpress still uses `jQuery 1.x` for fear of wrong compatibility with plugins and themes. They are trying to update it, but you can see that it will take a while yet

I am building a WP template where I have to use latest `Bootstrap 4`. The problem is that `Bootstrap 4` uses `jQuery 3.5.1`

Will the latest Bootstrap work properly with the default Wordpress `jQuery 1.12.4-wp` ?

**Comment by Johann-S** [comment, marked as answer]
Bootstrap 4 is compatible with jQuery v1.9.x to v3 see: https://github.com/twbs/bootstrap/blob/v4-dev/package.json#L91

Bootstrap 5 can work with the same version of jQuery but can work without jQuery too
